app/src/sass/_Landing.scss

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway:300i,500,700,800');
```

Comment:
Where do you fall in the inline styling, CSS in JS debate? Is there a solution in there that would make this more manageable for you? I honestly don't think this is too bad and I think the `#landing` is a fairly harmless solution.